### DISCUSSION TITLE
Raspberry Pi Camera V3?

### DISCUSSION THREAD

**Discussion by aBugMansLife** [interactionresearchstudio/CameraServer#107] [Q&A] [answered]
Hello,
I would like to find out whether or not the Pi Camera V3 NoIR will work with My Nature Watch?

If it doesn't work at this stage do you plan on adding support for it in future versions? 

Thanking you in anticipation.

**Comment by GaviG3** [comment]
I have the camera 3 without noir, I’m not able to get it to work, I get the attached error, not sure if it’s related.
![IMG_9612](https://github.com/interactionresearchstudio/NaturewatchCameraServer/assets/132746834/96b8505c-3c8a-424e-931b-b61536383f9e)

**Comment by andysheen** [comment, marked as answer]
Hello @aBugMansLife @GaviG3 ,
MyNaturewatch Camera doesn't currently support Pi Camera Module 3 as it requires the use of [PiCamera2](https://github.com/raspberrypi/picamera2 ) instead of PiCamera. PiCamera2 is currently in beta release, so we're a little hesitant to change over right now.
